terraform/overrides/nodes_use_ephemeral_storage_override.tf

```diff
@@ -0,0 +1,30 @@
+resource "openstack_compute_instance_v2" "node_server" {
+  block_device {
```

Comment:
I don't think that this is needed if the instance is boot-from-image. IIUC it would create a second volume in addition to the root disk. If you drop this, creating two additional blank devices should be possible.

Reply:
Does not work without this:

```
│ Error: Error creating OpenStack server: Bad request with: [POST https://api.zone-1.pebblestones.cloud:8774/v2.1/servers], error message: {"badRequest": {"code": 400, "message": "Block Device Mapping is Invalid: Boot sequence for the instance and image/block device mapping combination is not valid."}}
```